### ISSUE TITLE
[Bug]: Cannot remove additional storage from the workbench configuration

### ISSUE THREAD

**Issue by guimou** [opendatahub-io/odh-dashboard#1003]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Once you have created additional storage and assigned it to a workbench you cannot:
- Remove it from the workbench configuration. You have to go to the storage configuration to remove it.
- Edit the mountpoint, either from the workbench or the storage edit. You can only delete the mount and recreate it.

This is all totally unintuitive.

### Expected Behavior

You should be able to remove, or edit mountpoint, of additional storage directly in the workbench configuration.

### Steps To Reproduce

- Create additional storage
- Use it in a workbench
- You cannot remove it from the workbench configuration, nor modify the mountpoint from anywhere.

**Comment by andrewballantyne** [comment]
@guimou you're referring to the section in the expanded row of the workbench -- you can add one there but cannot remove one from there?

**Comment by guimou** [comment]
Yes, you can only add from there. And you cannot edit the mount point from anywhere, you have to delete/recreate, no place for typos :wink:
